assets/src/styles/blocks/Columns.scss

```diff
@@ -234,7 +242,7 @@
       }
 
       .step-info {
-        h5 > a {
+        h5 > a _-- {
```

Comment:
I kept this one a shorthand because it's not actually used in any lives campaigns. Plastics currently uses it, but Ana will be converting the usages to another style.

Eventually we'll probably want to remove this and unify the style (and heading level) with the other Columns styles.